Tools/CMake/AMReXParallelBackends.cmake

```
@@ -192,19 +194,27 @@ if (AMReX_HIP)
 
    set(CMAKE_MODULE_PATH "${HIP_PATH}/cmake" ${CMAKE_MODULE_PATH})
 
-   find_package(HIP REQUIRED)
+   find_package(hip REQUIRED)
 
-   if("${HIP_COMPILER}" STREQUAL "hcc")
-      message(FATAL_ERROR "Using (deprecated) HCC compiler: please update ROCm")
-   endif()
-
-   if(HIP_FOUND)
-      message(STATUS "Found HIP: ${HIP_VERSION}")
-      message(STATUS "HIP: Platform=${HIP_PLATFORM} Compiler=${HIP_COMPILER}")
+   if(hip_FOUND)
+      message(STATUS "Found HIP: ${hip_VERSION}")
+      message(STATUS "HIP: Platform=${hip_PLATFORM} Compiler=${hip_COMPILER}")
```

Compiler is still named: 
```suggestion
      message(STATUS "HIP: Platform=${hip_PLATFORM} Compiler=${HIP_COMPILER}")
```

Platform is not clear